doc/README.md

```diff
@@ -0,0 +1,146 @@
+# Apache Avro website
+
+This is a repository of Apache Avro website. The repository of Apache Avro can be found [here](https://github.com/apache/avro).
+
+This website is base on [Hugo](https://gohugo.io) and uses [Docsy](https://www.docsy.dev/) theme.
+
+## Getting started
+
+Clone this repository:
+
+```bash
+git clone --recurse-submodules https://github.com/apache/avro-website.git
```

Comment:
Do you think it would be OK to just remove this README.md for the moment?  I've done a couple full "pretend" releases to see how it would work with the existing build and release process, but it's pretty fussy for the moment (requiring moving in and out of the ubertool docker) and needs some work.

In general, these instructions are pretty out-of-date because they rely on one "big bang" deploy as opposed to adding per-release artifacts.

Reply:
Agreed!